src/main/scala/PrivateHome/id.xml

```diff
@@ -1 +1 @@
-<ids></ids>
\ No newline at end of file
+<ids/>
```

Comment:
I think this is incorrect! Use this.

<ids></ids>

IntelliJ also did like this ...